src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -79,13 +25,13 @@
                                     (merge item {:size size})]
 
       (:communities :collectibles) [fast-image/fast-image
-                                    {:source (:source item)
+                                    {:source (or (:source item) item)
                                      :style  {:width         size
                                               :height        size
                                               :border-radius border-radius}}]
 
       (:tokens :network :dapps)    [fast-image/fast-image
-                                    {:source (:source item)
+                                    {:source (or (:source item) item)
```

Comment:
I guess item can be a map and at the same time it can be a value (i guess a string). but this is not safe to do, I'd suggest to just be one type of thing.

The problem is we could get an exception while trying to extract a key from a string if we attempt:  `(:source "hey")`

Reply:
Yes, @ulisesmac. The item can be a map or number (for local resources) or string (for URL). 

> The problem is we could get an exception while trying to extract a key from a string if we attempt: (:source "hey")

We won't get any exceptions. The form `(:source "hey")` will produce `nil`. And since we use the `or` method, it will fall back to `item`.

Reply:
Oh, ok, it seems Clojurescript is pretty flexible sometimes.

Thanks!